internal/c/libqb.cpp

```diff
@@ -27356,13 +27356,19 @@ void sub__icon(int32 handle_icon, int32 handle_window_icon, int32 passed) {
 
 int32 func_screenwidth() {
 #ifdef QB64_GLUT
-    while (!window_exists) {
-        Sleep(100);
-    }
+    //while (!window_exists) {

```

Comment:
Don't leave commented code in the file :) If someone wants to see what was there before they can check the `git` history.